### ISSUE TITLE
Add subtitle lang

### ISSUE THREAD

**Issue by 3x1t-5tyl3** [GregTechCEu/GregTechCEu-1.19#1398]
### Feature Description

Currently subtitles are missing and they're a really nice accessibility feature.

![bxEiRhY9U0WCNIt6](https://github.com/GregTechCEu/GregTech-Modern/assets/61988002/ddd7ce9d-5e2d-4a28-9825-c742f0684bf4)

**Comment by htmlcsjs** [comment]
The lang keys exist in [1.12](https://github.com/GregTechCEu/GregTech/blob/a68d66f38ed40bc3dbe3527663566fa0c8b83421/src/main/resources/assets/gregtech/lang/en_us.lang#L6053) so it should be a relatively easy port

**Comment by 3x1t-5tyl3** [comment]
> The lang keys exist in [1.12](https://github.com/GregTechCEu/GregTech/blob/a68d66f38ed40bc3dbe3527663566fa0c8b83421/src/main/resources/assets/gregtech/lang/en_us.lang#L6053) so it should be a relatively easy port

Yeap. Working on it :P Though I question if I should update them?
Items assembling
Items separating
Items compressing
While these work. they could be more distinct.

Edit: some examples like these above
Assembler constructing
Centrifuge humming
Compressor thumping

Edit N°2: Seems like certain blocks also using other sounds.. like the sifter using the block place sound for sand? I wonder if there's a way to give it it's own entry while still using other sounds..
